app/styles/_thtrm-topic-box.scss

```diff
@@ -77,9 +76,28 @@
 }
 
 .thtrm-topic-list-item {
+  color: #fff;
   line-height: 28px;
   .material-icons {
     vertical-align: -20%;
     margin-right: 0.3em;
   }
 }
+
+.thtrm-topic-list-item-wide {
+  padding-left:1em;
+}
+
+
+.topic_gradient_ltr {
+  @include hGradient(#4f666d,#6a8389);
+}
+
+.topic_gradient_middle {
+  background-color : #6a8389;
+}
+
+.topic_gradient_rtl {
```

Comment:
I know we're not super good in naming CSS classes in this project but can we use at least dashes instead of underscores? :)